variables.tf

```
@@ -31,6 +31,12 @@ variable "workers_role_arns" {
   default     = []
 }
 
+variable "eks_cluster_service_role" {
```

```suggestion
variable "eks_cluster_service_role_arn" {
```

Please rename the variable and update the description.
Since we use only the ARN, no need to provide the entire object to the module (the `type = string` is not correct in this case)

---

Ok.  Commited your change and adapted the rest of the changes to conform.  This look ok now?

Sorry it took so long.